### PR TITLE
Use shortened symlink in CI to prevent character limit

### PR DESCRIPTION
**Pull request by eidenyoshida**

### CODE REVIEW

**Comment by AlexBrownAMD** [comment]
Looks like the rhel8 build has a new problem with the path. Also the cs9 build says py36 is missing.

**Comment by AlexBrownAMD** [comment]
I think it's failing cause normal current working directory is build instead of project root. This patch should fix it:

[cipath.txt](https://github.com/ROCmSoftwarePlatform/Tensile/files/8807202/cipath.txt)
.

**Review by benjaminulmer — DISMISSED** [review]
I'm not sure what the status of this PR is. If it's needed, we can probably get approval and merge since it's only jenkins changes. Until such a time, I'm blocking until after feature complete merge.

**Pull request closed without merging**
